internal/controller/persistentvolume/persistentvolume.go

```diff
@@ -171,7 +175,25 @@ func (r ReconcilePersistentVolume) reconcilePV(ctx context.Context, obj runtime.
 	}
 	defer cr.DeleteCredentials()
 
-	rbdVolID, err := rbd.RegenerateJournal(pv.Spec.CSI.VolumeAttributes, volumeHandler, requestName, cr)
+	var (
+		claimName      string
+		claimNamespace string
+	)
+	if pv.Spec.ClaimRef != nil {
+		claimName = pv.Spec.ClaimRef.Name
+		claimNamespace = pv.Spec.ClaimRef.Namespace
+	} else {
+		claimName = pv.Spec.CSI.VolumeAttributes[pvcNameKey]
+		claimNamespace = pv.Spec.CSI.VolumeAttributes[pvcNamespaceKey]
+	}
```

Comment:
we should not get pvc name and namespace form volumeattributes. this will be removed as part of https://github.com/ceph/ceph-csi/issues/2835